feat: added lab to build a quickstart (#1686)
newrelic/developer-website: src/markdown-pages/instant-observability/build-a-quickstart/create-a-quickstart.mdx

![graphiql interface](../../../images/instant-observability/graphiql.png)

Building a query is simple in graphiql. Check the appropriate boxes to build the query in Graphql. Your query should be same as follows:

>```json
>{
  actor {
    account(id: {_your account id_}) {
      alerts {
        nrqlCondition(id: "{_nrql conditions's id_}") {
          ... on AlertsNrqlStaticCondition {
            id
            name
            nrql {
              query
            }
          }
          terms {
            operator
            priority
            threshold
            thresholdDuration
            thresholdOccurrences
          }
        }
      }
    }
  }
}
>

Running the above query in Graphql will respond with the details of your alert condition that you can use to update your quickstart.

![cache hit ratio in graphql](../../../images/instant-observability/cache-hit-ratio-graphql.png)

For static condition, take the static-alert.yml file and start populating it using the information returned from above query. 

> First, name the file as **Cache Hit Ratio.yml** and populate it as follow:

Reviewer: ```suggestion
First, name the file _CacheHitRatio.yml_ and populate it:
```

A few suggestions here:

1. Remove some unnecessary words
2. Use italics for filenames
3. Don't have spaces in file names (you justified this already, but still a suggestion)

Author: I named the files with spaces to match the file name style for alerts in other quickstarts. elastic search quickstart named alerts as "Cluster health.yml", "Index Health.yml". @jaesius what's your opinion?

## Contribute quickstart to GitHub

Your quickstart is now ready to be published. You're going to commit your changes back to GitHub where it will be reviewed by New Relic. 
Follow the [conventional commit syntax](https://github.com/newrelic/newrelic-observability-packs/blob/alerts-doc-update/CONTRIBUTING.md#using-conventional-commits) for New Relic to commit your changes.

```bash
 git add -A
 git commit -m "feat(flashdb): Added a quickstart for flashdb"
```

<Callout variant="important">

We, at New Relic, are always happy to receive contributions from the community. However, we are expecting a number of contributions for flashdb following the same lab course. 
To help us manage your contributions efficiently, we encourage you to label your contribution following this particular lab as **lab** using `git tag` command.

</Callout>

Push your changes to GitHub:
```bash
 git Push
```

Once you've pushed your changes to GitHub, you can [create a pull request](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/creating-a-pull-request).

![pull request](../../../images/instant-observability/pull-request.png)

> Submit and wait for the review. Thanks a lot for your submission!

Reviewer: ````
<Callout variant="important">

At this stage, your quickstart will be automatically validated in GitHub. If this were a real-life submission, it would then be approved and incorporated into New Relic I/O for your users to see. As this is a lab, we will not actually review this PR. Feel free to delete it after it validates your work.

</Callout>
````

Just to help them understand more about the validation in GitHub actions.

> ## What's Next

Reviewer: ```suggestion
## Homework
```

For these labs, we wanted to call this section "Homework" to encourage self-motivated future action.

> If you haven't already, we encourgae you to read through the related docs as follow:

Reviewer: ```suggestion
If you haven't already, we encourage you to read through some related docs:
```

- [Dashboards](https://docs.newrelic.com/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards/)
- [Alerts](https://docs.newrelic.com/docs/alerts-applied-intelligence/#:~:text=Together%2C%20New%20Relic%20Alerts%20and,metrics%20you%20care%20about%20most.)

> The above docs will give you a head start on your observability journey with New Relic. We always welcome contributions from community so feel free to get in touch.

Reviewer: ```suggestion
The above docs give you a head start on your observability journey with New Relic. We always welcome contributions from community so feel free to get in touch.
```

  - Cache hits: Monitor the cache hits
  - Errors: Monitor the errors occured for each type of database transaction
  - Database keys: Monitor the keys for database
 
  Alerts:
  - Cache hit ratio: Alert is triggered when cache hit ratio falls below the set limit.
  - Read response: Alert is triggered when read responses are slow and exceeds a set limit

summary: |
  Flashdb is the fastest database in the space. 
  This quickstart allows you to get visibility into the performance of flashdb.

# Support level: New Relic | Verified | Community (required)
level: Community

# Design
icon: icon.jpeg
logo: logo.png

# Authors of the pack (required)
authors:
  - {"enter your name"}

# References to other packs (optional)
# References are grouped by the type: operating-system, database, webserver, queue, ..
# The reference under the type is a at least one system, so at least one operating-system, database, ..
references:
  - type: database
    name:
    - flashdb
```

Here's how your final _flashdb_ quickstart folder should look like.
![flashdb quickstart directory](../../../images/instant-observability/final-flashdb-quickstart.png)

> ## Test your quickstart

Reviewer 2: While the import script is not working, let's remove this section. Once the import script functions appropriately, it can be saved in a branch and added back into the lab.
